client/src/pages/user/Profile.js

```diff
@@ -113,8 +168,13 @@ const Profile = () => {
                     onChange={(e) => setAddress(e.target.value)}
                     className="form-control"
                     id="exampleInputEmail1"
+                    required
```

Comment:
as discussed, we can use placeholder (required) to tell users this field cannot be empty 👍

Reply:
ok i will add this in